modules/runners/lambdas/runners/src/scale-runners/scale-up.ts

```diff
@@ -169,16 +169,17 @@ async function GetRunnerTypes(org: string, repo: string, enableOrgLevel: boolean
   console.debug(`[GetRunnerTypes] Grabbing runnerTypes`);
 
   const githubAppClient = await createGitHubClientForRunner(org, repo, enableOrgLevel);
+  // For org enabled runners we use ${org}/.github as the repository to hold the singular configuration
+  const configRepository = enableOrgLevel ? '.github' : repo.split('/')[1];
```

Comment:
Up for debate on whether or not we actually want to use the `pytorch/.github` repository as our place to store this configuration.

I'm also fine with this going into `pytorch/test-infra`